### ISSUE TITLE
Error on installation of Directus

### ISSUE THREAD

**Issue by SvenBudak** [directus/directus#2514]
I tryed yesterday for first time to install directus. The first thing is that the documentation is not working well. Some menu entries dont load =)

Then i find over google: https://docs.directus.io/advanced/api/configuration.html#project-config-files

I tryed first the manual web installer: https://directus.upperyard.de/admin/#/install But it shows:

```
Failed to load resource: the server responded with a status of 503 ()
TypeError: Cannot read property 'server' of undefined
```

so in did the manuall config thing. I copy all from example.php to test.php and edited `name`, `username` and `password`. When o try to open now https://directus.upperyard.de/ i get redirected to `/admin/#/login` so far so good... but here i see a error: `Request failed with status code 500`:

![image](https://user-images.githubusercontent.com/4930169/71006864-52af2b80-20e6-11ea-9245-9ee816379925.png)

And in the Console i see:

```
VM198:1 GET https://directus.upperyard.de/test/ 500
6VM198:1 GET https://directus.upperyard.de/server/ping 500
```

Then i tryed to do the `Configure with Script` thing...

```
bin/directus install:config -n d4329433 -u d0f0c433 -p hYZSDFzsfPzxM3RFSDZH2
```

and get this error message in bash:

```
PHP Warning:  file_get_contents(/www/htdocs/w01a128d/directus/config/__api.json): failed to open stream: No such file or directory in /www/htdocs/w01a128d/directus/src/core/Directus/Console/Modules/InstallModule.php on line 135

Warning: file_get_contents(/www/htdocs/w01a128d/directus/config/__api.json): failed to open stream: No such file or directory in /www/htdocs/w01a128d/directus/src/core/Directus/Console/Modules/InstallModule.php on line 135
Creating config files required: db_name, db_user, super_admin_token
```

What i did wrong?

**Comment by rijkvanzanten** [comment]
Heya! Could you confirm your server meets all [the requirements](https://docs.directus.io/getting-started/installation.html#requirements) of Directus?

The error you got initially indicates to me that Directus was trying to do something it doesn't have access to. Maybe you forgot to give it write access to its folders?

**Comment by SvenBudak** [comment]
The requirements are all given.

but i dont change folder permissions because i didnt saw any in dokumentation about this. i need to change permissions?

**Comment by rijkvanzanten** [comment]
Yeah, the PHP user needs to have permission to write to the folders, see https://docs.directus.io/getting-started/setup-checklist.html#checklist

**Comment by benhaynes** [comment]
Also: https://docs.directus.io/advanced/requirements.html#permissions

**Comment by SvenBudak** [comment]
Hmmm i changed the owner to www-data but still same...
Is there a composer installer somewhere that configures everything automaticly?

**Comment by lazarporter** [comment]
@benhaynes Both of those links are giving me a 404.  Have those resources moved or is that a temporary issue?

**Comment by benhaynes** [comment]
Yes, we've been working on updating the docs... VuePress doesn't keep track of migrated links/URLs.

https://docs.directus.io/getting-started/troubleshooting.html#install-checklist
https://docs.directus.io/installation/git.html#step-4-set-read-write-permissions

**Comment by mescrich** [comment]
I'm having the same problem, i'm installing directus on a docker container without enviroment variables but when trying to install:config I'm getting this error: 
```
Warning: require(/var/directus/config/.php): failed to open stream: No such file or directory in /var/directus/src/core/Directus/Util/Installation/InstallerUtils.php on line 96

Fatal error: require(): Failed opening required '/var/directus/config/.php' (include_path='.:/usr/local/lib/php') in /var/directus/src/core/Directus/Util/Installation/InstallerUtils.php on line 96
```
any suggestions?

**Comment by tehuel** [comment]
Got the same problem when installing on a shared hosting. In my case, solved it by changing the PHP version from 7.1 to 7.4

**Comment by rijkvanzanten** [comment]
Closing this as a generic 500 can be caused by a range of different server related issues. If you run into the same problem, feel free to open a new issue detailing your specific configuration and environment 🤗
